functional-test/src/test/groovy/org/rundeck/util/container/BaseContainer.groovy

```diff
@@ -570,4 +571,39 @@ abstract class BaseContainer extends Specification implements ClientProvider, Wa
         waitForRundeckAppToBeResponsive()
     }
 
+    List<String> getAllLogs(String execId, Function<Map, String> paramGen, int max=20) {
```

Comment:
this method also exist in "ExecutionOutputSpec" class and "ProBaseContainer" class. It would be good to mark those as deprecated and just use this new one instead